### PR TITLE
[14.0] [mig] sale_order_line_price_history

### PR DESCRIPTION
**Pull request by dsolanki-initos**
Supersedes #1447

### CODE REVIEW

**Comment by dsolanki-initos** [comment]
@CasVissers-360ERP 
It would be nice if you can review this PR

**Comment by dsolanki-initos** [comment]
It would be nice if we could get the reviews here

**Review by rousseldenis — APPROVED** [review]
Code review

File: sale_order_line_price_history/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_sale_order_line_price_history,sale_order_line_price_history,model_sale_order_line_price_history,base.group_user,1,1,1,1
```
Comment (pedrobaeza) [inline sale_order_line_price_history/security/ir.model.access.csv]: Please give permission only to sales users.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Can you please switch pre-commit and migration commits order?

**Comment by pedrobaeza** [comment]
Can you please check Travis?

**Comment by dsolanki-initos** [comment]
Yes I checked but error is not due to this module it is from the module 'sale_product_seasonality'

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@simahawk can you please take a look to the failing test on `product_seasonability`?

**Comment by simahawk** [comment]
I just merged the fix here https://github.com/OCA/sale-workflow/pull/1641 (last commit) :wink: 
Can you rebase?

**Comment by pedrobaeza** [comment]
Thanks both:

/ocabot merge nobump

**Pull request merged**